deeplearning4j-scaleout/spark/dl4j-spark/src/main/scala/org/apache/spark/TaskContextHelper.scala

```diff
@@ -0,0 +1,11 @@
+package org.apache.spark
+
+
+/**
+  * This simple helper is used to get access to package-protected Scala TaskContext.setTaskContext method
```

Comment:
I would like to see an explicit reference to https://issues.apache.org/jira/browse/SPARK-18406 in comments here.